### DISCUSSION TITLE
Virtual Tranining를 듣고난후...

### DISCUSSION THREAD

**Discussion by phphll** [fusiondevkr/pl900#45] [1 Q&A]
바우처 관련된 메일이 안오는데 음....

저만 안오는건지 궁금하네요 ㅠㅠ

**Comment by justinyoo** [comment]
이게 바로 오진 않고 한 2-3일 지나서 이메일로 도착한다고 하네요? 저도 아직 못 받았습니다.

**Reply by sesme100** [reply]
저도 아직 이메일을 못 받았는데요. Virtual Training 관리자분께 한번 연락 해볼게요.

**Reply by phphll** [reply]
지금 확인해 보니 날라왔네요 혹시나 하는 마음에! ㅎㅎ 답변감사드립니다👍

**Reply by phphll** [reply]
아 혹시 개발자 계정으로 시험신청해도 되나요 ?

**Reply by phphll** [reply]
개인 계정이 나으려나요 .... 궁금하네요

**Reply by justinyoo** [reply]
신청한 이메일 주소로 하셔야 하지 않을까요?
